x/mint/minter.go

```diff
@@ -7,45 +7,53 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// current inflation state
+// Minter represents the minting state
 type Minter struct {
-	InflationLastTime time.Time `json:"inflation_last_time"` // block time which the last inflation was processed
-	Inflation         sdk.Dec   `json:"inflation"`           // current annual inflation rate
+	LastUpdate       time.Time `json:"last_update"`       // time which the last update was made to the minter
+	Inflation        sdk.Dec   `json:"inflation"`         // current annual inflation rate
+	AnnualProvisions sdk.Dec   `json:"annual_provisions"` // current annual expected provisions
 }
 
-// minter object for a new minter
-func InitialMinter() Minter {
+// Create a new minter object
+func NewMinter(lastUpdate time.Time, inflation,
+	annualProvisions sdk.Dec) Minter {
+
 	return Minter{
-		InflationLastTime: time.Unix(0, 0),
-		Inflation:         sdk.NewDecWithPrec(13, 2),
+		LastUpdate:       lastUpdate,
+		Inflation:        inflation,
+		AnnualProvisions: annualProvisions,
 	}
 }
 
+// minter object for a new chain
+func InitialMinter(inflation sdk.Dec) Minter {
+	return NewMinter(
+		time.Unix(0, 0),
+		inflation,
+		sdk.NewDec(0),
+	)
+}
+
+// default initial minter object for a new chain
```

Comment:
I would add the default inflation to the godoc here.

Reply:
added